build.gradle.kts

```diff
@@ -59,7 +59,7 @@ tasks.withType<org.liquibase.gradle.LiquibaseTask>().configureEach {
     @Suppress("MAGIC_NUMBER")
     this.javaLauncher.set(project.extensions.getByType<JavaToolchainService>().launcherFor {
         // liquibase-core 4.7.0 and liquibase-gradle 2.1.1 fails on Java >= 13 on Windows; works on Mac
-        languageVersion.set(JavaLanguageVersion.of(11))
+        languageVersion.set(JavaLanguageVersion.of(13))
```

Comment:
what about message above? and if all is fine, why `13`? and not `17`, at least?

Reply:
it fails on Windows, I guess @Cheshiriks uses `Mac`

Reply:
I accidentally committed this. Rolled back.